resources/texts/change_log.txt

```diff
@@ -1,3 +1,73 @@
+[5.0]
+Beta
+<i><a href="https://youtu.be/kRj7pR4OkQA">Watch the launch event</a> to learn more about Ultimaker Cura 5.0 beta.</i>
+
+* New slicing engine
+Following special beta releases to test the Arachne engine, we are pleased to announce our new slicing engine is here in Ultimaker Cura! This all-new engine uses variable line widths when preparing files for printing, meaning you can now print thin and intricate parts more accurately and with greater strength. 
+
+* Renewed the Ultimaker Cura Marketplace
+We have streamlined the workflow for accessing the Ultimaker Marketplace inside of Ultimaker Cura. The UI has been improved and it’s now easier and faster to find and install plugins and material profiles.
+
+* Improved print profiles for Ultimaker printers
+The new slicing engine in Ultimaker Cura 5.0 beta has helped us to improve our print profiles. This means that users of Ultimaker printers can achieve speed increases of up to 20%.
+
+* Other new features and improvements:
+- Upgrade from Qt5 to Qt6
+- New Cura icon
+- New Cura splash screen
+- Fuzzy skin works with the new engine
+- Connect Top/Bottom polygons work with the new engine
+- Updated the digital build plates for Ultimaker printers
+- Introduce Minimum Wall Line Width, contributed by @BagelOrb
+- Improved description and label of wall Transition Angle, contributed by @BagelOrb
+- Settings for metal printing implemented
+- Shrinkage compensation is now available for PLA, tPLA and PETG
+- Improved default Line Widths for Spiralize
+- Decrease resolution to remove some buffer underruns
+- Removed the setting Center Last from Wall Ordering
+- Increased Filter Distance
+- Hide the Variable Line Strategy in UI
+- Incomplete languages are now shown in the language drop-down menu
+
+* Bug fixes:
+- Added the Scale Fan Speed From 0 to 1 setting for printers that interpreted fan speed as percentages
+- Fixed a bug with extra travel moves increased the printing time, contributed by @BagelOrb
+- Fixed a bug where Monotonic Ironing breaks Ironing, contributed by @BagelOrb
+- Changed the priority of CuraEngine
+- Fixed a bug where increasing Filter distances creates extremely wide lines, contributed by @BagelOrb
+- Fixed double scroll bar, contributed by @fieldOfView
+- Fixed a bug where maximum resolution/deviation was not applied to surface mode, contributed by @BagelOrb
+- Fixed a bug where the seam placement was uneven
+- Fixed a bug where Top Surface Skin Layers didn't work
+- Updated the Tinyboy Fabricator printer definitions, contributed by @reibuehl
```

Comment:
Some of the updated profiles are here and some under printers ... I could see it both ways, but we should maybe pick one.